modules/custom-status/custom-status.php

```diff
@@ -453,6 +442,69 @@ public function remove_or_add_publish_capability_for_user( $allcaps, $cap, $args
 		return $allcaps;
 	}
 
+	// ToDo: Once custom status has been refactored, the below CRUD methods should be unified with the
```

Comment:
This was my little todo to create a generic DB utility class that could handle CRUD operations rather than us repeating the code everywhere.

Reply:
Agree! This would be a great way to pull EM and custom-status meta getters and setters into separate classes.

Reply:
That being said, let's create a separate issue for this `ToDo` and ideally remove all of the `ToDo`s in this PR.

Reply:
Yip, I was gonna do that before a merge just so all the issues are made in one go. 100% agreed though